components/organisms/ContributorProfilePage/contributor-profile-page.tsx

```diff
@@ -73,21 +75,25 @@ const ContributorProfilePage = ({
   const prsMergedPercentage = getPercent(prTotal, prMerged || 0);
   const isLoaded = !loading && !error;
 
+  const { bio, location, name, twitter_username } = user || {};
```

Comment:
if you put a different error container before this line, you will not need to `|| {};` at the end, as `user` would always be inferred when loaded.